src/Player.java

```diff
@@ -0,0 +1,45 @@
+/**
+ * Created by strejolopez on 11/29/16.
+ */
+public class Player {
+    private String name;
+    private String token;
+    private int money;
+
+    public Player(String name, String token) {
+        this.name = name;
+        this.token = token;
+        this.money = 1500;
+    }
+
+    public String getName() { return this.name; }
+    public String getToken() { return this.token; }
+    public int getMoney() { return this.money; }
+
+
+    public void payMoney(int amount){
+        if(money < 1500) {
+            this.money -= amount;
+        System.out.println();
+    }
+    else{
+            System.out.println(this.money-= amount);
```

Comment:
Ok you have the same code `this.money -= amount;` in both the if and the else.  Also, you have a println with nothing in it.  Additionally, don't put math or assignments in the parenthesis if you can avoid it.